SIDECAR_RELEASE_PROCESS.md

```diff
@@ -48,24 +49,29 @@ naming convention `<hostpath-deployment-version>-on-<kubernetes-version>`.
    Kubernetes versions that are no longer supported.
 
 ## Release Process
+
 1. Identify all issues and ongoing PRs that should go into the release, and
   drive them to resolution.
 1. Download v2.8+ [K8s release notes
   generator](https://github.com/kubernetes/release/tree/master/cmd/release-notes)
 1. Generate release notes for the release. Replace arguments with the relevant
   information.
     * For new minor releases on master:
-        ```
+
+      ```shell
```

Comment:
Probably should remove indention here, too.

Reply:
Done.